Ga event label consolidation (#22992)
department-of-veterans-affairs/va.gov-team: teams/vsp/teams/insights-analytics/ga-events-data-dictionaries.md

| Product Area Used - Other | 5 | User | Distinguish users who have ever completed at least one transaction in the Other product area |
| Product Used - Education Forms | 6 | User | - |
| Product Used - Health Care Application | 7 | User | - |
| Product Used - Claims/Appeals Tracker | 8 | User | - |
| Product Used - Secure Messaging | 9 | User | - |
| Product Used - Rx Refills | 10 | User | - |
| Product Used - VA Letters | 11 | User | - |
| Product Used - Post 9/11 GI Bill Status | 12 | User | - |
| Product Used - Facility Locator | 13 | User | - |
| Product Used - GIBCT | 14 | User | - |
| Product Used - Health Records | 15 | User | - |
| Product Used - Burial Forms | 16 | User | - |
| Product Used - Pension Forms | 17 | User | - |
| Product Used - VIC | 18 | User | - |
| Product Used - Pre-need Burial Application | 19 | User | - |
| Product Used - Discharge Upgrade | 20 | User | - |
| Logged In | 21 | Session | Distinguish users who have logged in |
| LOA Level | 22 | Session | Level of authorization - collected right after a user logs in  |

> | Telephone Contact | 23 | User | User's phone contact information collected after a user logs in |

Reviewer: @laurensimpson14 This gave me a bit of a scare at first 😅 , this is not the user's log in...this is the telephone number of when a phone number is clicked on

| Education 22-0994 Currently Working Industry | 47 | Hit | User response to the currently working industry question (yes or no) collected on the edu-0994-submission-successful event label on the www.va.gov/education/about-gi-bill-benefits/how-to-use-benefits/vettec-high-tech-program/apply-for-vettec-form-22-0994/review-and-submit/ page |
| Education 22-0994 Made Program Selection | 48 | Hit | User response to the made program question (yes or no) collected on the edu-0994-submission-successful event label on the www.va.gov/education/about-gi-bill-benefits/how-to-use-benefits/vettec-high-tech-program/apply-for-vettec-form-22-0994/review-and-submit/ page |
| Education - STEM Scholarship Applicant | 49 | Hit | User response to STEM scholarship question (yes or unanswered) collected with the edu-howToApply-applyNow event label |
| Profile Section Saved - Direct Deposit | 50 | User | User who is logged in saved their direct deposit information |

> | Health Facility Location | 51 | Hit | Name of hte health facility location that the event/pageview/action is taking place (for example, Pittsburgh on this page: www.va.gov/pittsburgh-health-care/) |

Reviewer: Just a type change from hte to the

> | Health Facility Type | 52 | Hit | This CD is set as `Campus` when a user visits or completes actions on this page:         <br>www.va.gov/careers-employment/vetsuccess-on-campus/  |

Reviewer: Perhaps we consider splitting out a 4th "example" column and keep this column as strictly a definition....your specific pages are great and we should utilize those as examples

| Error Key | 57 | Hit | An error occured (for example, 503 internal error if the disability-ratings-list-load-failed event happens on this page: www.va.gov/disability/view-disability-rating/rating/) |
| Profile Section Saved - SMS Optin and Optout | 58 | User | User who is logged in saved their SMS opt-in and opt-out information |
| Education - Type of Education | 59 | Hit | **No data is being collected** |
| Product - Disability Ratings Viewed | 60 | User | Value is set to true when a user views disability ratings |
| Video Title | 61 | Hit | Title of a video (for example, there is a video titled `How to Help Homeless Veterans on this page: www.va.gov/homeless/) |
| Product - Health Care VAOS User | 62 | User | A value set to true if a user is logged in as a health care VAOS user |
| Product Area Used - Disability | 63 | User | Distinguish users who have ever completed at least one transaction in the Disability product area |
| Education - Benefit Update | 64 | Hit | User updates their benefits within the education benefit hub (values for this CD include (1) new (2) update (3) stem-scholarship) |

> | Education - Benefit Claim for Self | 65 | Hit | User's response to the question of if they are filling out the benefit claim for themselves. Options include: (1) yes (2) no (3) unanswered  |

Reviewer: If we have an example of a form number we should include that here

| Education - Receiving Sponsor Benefits | 69 | Hit | User's response to the question of receiving sponsor benefits. Options include: (1) yes (2) no (3) unanswered. Found on this page: www.va.gov/education/eligibility/ as well as this page: www.va.gov/education/how-to-apply/ |
| Education - Sponsor Available | 70 | Hit | User's response to the question of sponsor available. Options include: (1) yes (2) no (3) unanswered. Found on this page: www.va.gov/education/eligibility/ as well as this page: www.va.gov/education/how-to-apply/ |
| Health Care VAOS - Appointment Type | 71 | Hit | Appointment type (either `confirmed` or `pending`) for a VAOS apointment |
| Health Care VAOS - Type of Care | 72 | Hit | Answer to the question asking users to `choose the type of care you need`. Options include: `primary care`, `mental health`, `optometry`, `COVID-19 vaccine`, etc. Pages this CD is set: www.va.gov/health-care/schedule-view-va-appointments/appointments/new-appointment |
| Health Care VAOS - Flow | 73 | Hit | Type of flow for a VAOS request. Options include: `va-request`, `direct`, and `cc-request` |

> | Education - 22-1995 - Exhausted All Benefits | 74 | Hit | **No data is being collected** |

Reviewer: If no data is being collected here let's open a bug and put in the backlog, low priority

| Health Care VAOS - Facility Type | 75 | Hit | Facility type a user selects in the VAOS flow. Options incude: `va` or `cc` |

> | Health Care VAOS - Reason for Appointment | 76 | Hit | **No data is being collected**. Answer to the question asking users to `choose a reason for your appointment`. Options include: `routine or follow-up visit`, `I have a new medical issue`, `I have a concern or question about my medication`, or `My reason isn't listed here`. |

Reviewer: We should look at this as part of the other vaos cleanup

| Education - YR - Filter - Contribution Amount | 79 | Hit | Contribution amount CD is set to either `(not set)` or `unlimited` |
| Education - YR - Filter - Number Of Students | 80 | Hit | Contribution amount CD is set to either `(not set)` or `unlimited` |
| Education - School Name | 81 | Hit | CD is set to the school name entered into the text box "Enter full school name" found on this page: https://www.va.gov/education/yellow-ribbon-participating-schools/. Many of the `Education - School Name` CD entries are set to `(not set)` |
| Education - School State | 82 | Hit | CD is set to the state entered into the box "State or territory" found on this page: https://www.va.gov/education/yellow-ribbon-participating-schools/. Many of the `Education - School Name` CD entries are set to `(not set)` |
| Education - School City | 83 | Hit | CD is set to the city entered into the box "City" found on this page: https://www.va.gov/education/yellow-ribbon-participating-schools/. Many of the `Education - School Name` CD entries are set to `(not set)` |

> | [old] Education - YR - # of Search Results | 84 | Hit | CD is set to an integer matching the number of search results of a search on this page: www.va.gov/education/yellow-ribbon-participating-schools/ |

Reviewer: If this is not actually collecting anymore let's change to unused in GA and mark this here

| VAOS Facility ID | 85 | Hit | Facility ID of the facility selected during the VAOS flow |
| Product Name | 86 | Hit | Product name set in the `bam-form-change` event label on this page: www.va.gov/health-care/order-hearing-aid-batteries-and-accessories/order-form-2346 |
| Product IDs | 87 | Hit | Product ID set in the `bam-form-change` event label on this page: www.va.gov/health-care/order-hearing-aid-batteries-and-accessories/order-form-2346 |
| Product IDs - Failed | 88 | Hit | Product ID failed on the order form 2346 |
| Partial Form Submission Failed | 89 | Hit | CD set to `1` if order form 2346 fails for being only partially complete |

> | Component Relative Size | 90 | Hit | CD set to `Small` on this page: www.va.gov/gi-bill-comparison-tool/profile/ |

Reviewer: ```suggestion
| Component Relative Size | 90 | Hit | A generic CD to capture a relative description of component size, used on accordions on this page: www.va.gov/gi-bill-comparison-tool/profile/ |
```

| Global - Service Branches | 101 | Hit | User indicated branch served in such as `army` or `navy`on the edu-1995 form  |
| Global - Service Start Dates | 102 | Hit | CD set to the date entered as the user's/vet's service start date on the edu-1995 form |
| Global - Service End Dates | 103 | Hit | CD set to the date entered as the user's/vet's service end date on the edu-1995 form |
| Education - Field of Study / Degree | 104 | Hit | User indicated field of study and length of time on the form 22-10203 form (for example, `sixMonthsOrLess` or `Computer Science`) |
| Education - Benefits Remaining | 105 | Hit | User indicated length of time for benefits remaining on the form 22-10203 form. Options include: `sixMonthsOrLess`, `none`, or `moreThanSixMonths` |
| Health Care VAOS - Express Care Reason | 106 | Hit | CD set to the express care reason a user selects while scheduling an appointment. Options include: `back pain`, joint or muscle pain or minor injury`, etc. Page to schedule an appointment: www.va.gov/health-care/schedule-view-va-appointments/appointments/new-express-care-request/additional-details/ |

> | Health Care VAOS - Express Number of Cards | 107 | Hit | **No data is being collected** |

Reviewer: _Let's open a bug if this is not collecting, again low priority_

```suggestion
| Health Care VAOS - Express Number of Cards | 107 | Hit | Number of express cards loaded |
```

> | Health Care VAOS - Upcoming Number of Cards | 108 | Hit | **No data is being collected** |

Reviewer: _Same here, let's open a bug if not collecting_

```suggestion
| Health Care VAOS - Upcoming Number of Cards | 108 | Hit | Number of upcoming appointment cards loaded |
```

| Facility Locator Search - Facility Type | 110 | Hit | Facility type (for example, `health` or `pharmacy`) collected from a facility locator search on this page: www.va.gov/facilities/ |
| Health Care VAOS - Returned Item Type | 111 | Hit | Returned item type (for example, `upcoming` or `video_home`) from the VAOS flow and collected on this page: www.va.gov/health-care/schedule-view-va-appointments/appointments/  |
| Health Care VAOS - Returned Number of Items | 112 | Hit | CD is set to an integer matching the number of items of a search on this page:         <br>www.va.gov/health-care/schedule-view-va-appointments/appointments/ |
| Global - Dependent Debt | 113 | User | CD is set to either `Without Dependent Debt` or `With Dependent Debt` depending on the user's selection on this page: www.va.gov/manage-va-debt |
| Type Ahead Options Keyword Selected | 114 | Hit | User selects a type ahead option. The values for this CD are endless. |
| Type Ahead Options List | 115 | Hit | User views type ahead options list |

> | Phased Out Product | 116 | Session | CD value is set to either `type ahead` or `chapter 36` |

Reviewer: _For context, this is our current solution for dealing with phased roll outs (where teams first roll out their products to a percentage of the total population)_

```suggestion
| Phased Out Product | 116 | Session | Description or name of product with a phased roll out |
```

| Type Ahead Option Chosen Position | 117 | Hit | CD value is set to the position of the type ahead option that the user selected (either `1`, `2`, `3`, `4`, `5`, or `(not set)`) |
| HLR - Same Office Requested | 118 | Hit | CD value is set to either `yes` or `no` based on the user's selection on the 20-0996 form |
| HLR - Informal Conference Requested | 119 | Hit | CD value is set to either `yes`, `yes-with-rep`, or `no` based on the user's selection on informal conference requested the 20-0996 form |
| Search Results Type | 120 | Hit | CD value is set to the search result type (`title`, `pdf`, `cta`, etc.) a user selects when completing a search on any search page ( /search/, /find forms/, /yellow-ribbon-participating-schools/, etc.) |

> | Medallia Feedback UUID | 121 | Session | The UUID for Medallia feedback typically found on this page: www.va.gov/facilities/index.html?XXX |

Reviewer: Can use this as an example, as that's where one of the main surveys is today 
```suggestion
| Medallia Feedback UUID | 121 | Session | The UUID for Medallia feedback survey, not available with all submissions |
```

| Medallia Survey ID | 122 | Hit | Survey ID for the Medallia survey that is being leveraged |
| Medallia Survey Status | 123 | Hit | The Medallia survey status is set to either `AlwaysOn`, `Intercept`, or `Invite` and is collected when a user begins a survey |

> | Placeholder - CLP Event Name | 124 | Hit | **No data is being collected** |

Reviewer: This should be getting set as part of the CLP work now 
```suggestion
| Placeholder - CLP Event Name | 124 | Hit | Name of event selected from campaign landing page |
```

| Medallia - Survey Contents | 125 | Hit | String of information collected when a survey is submitted or closed |

> | Education - Facility Code | 126 | Hit | This CD is most commonly set as `(not set)`, however a facility code is captured if a user enters a facility on the page: www.va.gov/education/yellow-ribbon-participating-schools/ |

Reviewer: ```suggestion
| Education - Facility Code | 126 | Hit | This CD is most commonly set as `(not set)`, however a facility code is captured if a user selects a search result option with a facility code|
```

> | Education - Institution ID | 127 | Hit | This CD is most commonly set as `(not set)`, however an institution ID is captured if a user enters the relevant information on the page: www.va.gov/education/yellow-ribbon-participating-schools/ |

Reviewer: _There's probably a good amount of cases where there is no institution ID, however we have their developers push this from the backend on search results click_

```suggestion
| Education - Institution ID | 127 | Hit | This CD is most commonly set as `(not set)`, however an institution ID is captured if a user selects a search result option with an institution ID
```

| Education - YR - School Search Result Details | 128 | Hit | Set as a long string of information containing a school's details from a school search result (such as city, country, state, etc.) from a search on the page: www.va.gov/education/yellow-ribbon-participating-schools/ |

> | Click URL | 129 | Hit | **No data is being collected** |

Reviewer: _This is now collecting_

```suggestion
| Click URL | 129 | Hit | CD consistently collects the click URL as a paper trail |
```

| Health Care VAOS - Preferred Language | 130 | Hit | When a user changes their preferred language option on this page: www.va.gov/health-care/schedule-view-va-appointments/appointments/new-appointment/review/. Most commonly this CD is set to `(not set)` |
| Health Care VAOS - Number of Preferred Providers | 131 | Hit | Set to either `1` or `0` depending on the user's selection of number of preferred providers when a user submits the VAOS form to schedule an appointment here: www.va.gov/health-care/schedule-view-va-appointments/appointments/new-appointment/review/ |

> | GTM Version Number | 132 | Hit | **No data is being collected**. CD value is set to the current GTM verison that is running on VA.gov. This is useful for checking historical data and knowing what the GTM configuration was for the period of time being analyzed.  |

Reviewer: ```suggestion
| GTM Version Number | 132 | Hit | CD value is set to the current GTM verison that is running on VA.gov. This is useful for checking historical data and knowing what the GTM configuration was for the period of time being analyzed.  |
```
_Now published and collecting_

> | Component Library Version | 133 | Hit | **No data is being collected**. |

Reviewer: ```suggestion
| Component Library Version | 133 | Hit | Component library package version number |
```

_This will be collecting very shortly, so let's just update here_

| Event Complete | 2 | Hit | Time |  |
| IDme Login Attempt | 3 | Hit | Integer | When a user attempts to login via IDme |
| IDme Login Success | 4 | Hit | Integer | When a user successfully logs in via IDme |
| MHV Login Attempt | 5 | Hit | Integer | When a user attempts to login via MHV |
| MHV Login Success | 6 | Hit | Integer | When a user successfully logs in via MHV |
| DSLogon Attempt | 7 | Hit | Integer | When a user attempts to login via DSLogon |
| DSLogon Success | 8 | Hit | Integer | When a user successfully logs in via DSLogon |
| Education - Count of Programs Saved | 9 | Hit | Integer | Occurs with a Vet Tec Completion with the edu-0994 form |
| File Uploads | 10 | Hit | Integer | When a user uploads a file |
| Disability Calculator Rows | 11 | Hit | Integer | Number of rows being used in the disability rating calculator found on /disability/about-disability-ratings/ |
| Unused | 12 | Hit | Integer | - |
| Unused | 13 | Hit | Integer | - |
| Unused | 14 | Hit | Integer | - |
| Unused | 15 | Hit | Integer | - |
| Unused | 16 | Hit | Integer | - |
| GIBCT - Profile Benefit Estimate | 17 | Hit | Currency (Decimal) |  |
| Unused | 18 | Hit | Integer | - |

> | Address Validation Needed | 19 | Hit | Integer | This is triggered with a profile navigation action |

Reviewer: ```suggestion
| Address Validation Needed | 19 | Hit | Integer | Increments by 1 each time address validation is needed |
```

> | Address Suggestion Provided | 20 | Hit | Integer | This is triggered with a profile navigation action |

Reviewer: _I kind of hate I know some of these things just by reading them_ might have done this work 😄 

```suggestion
| Address Suggestion Provided | 20 | Hit | Integer | Increments each time at least 1 address suggestion is provided at address validation |
```

| Address Suggestion Used | 21 | Hit | Integer | User leverages an address suggestion when submitting information on their profile |

> | Home Address Suggestion Failures | 22 | Hit | Integer | Can occur when there is a profile save failure |

Reviewer: ```suggestion
| Home Address Suggestion Failures | 22 | Hit | Integer | On profile save edit failure, this increments by 1 for home address specific failures |
```

> | Mailing Address Suggestion Failures | 23 | Hit | Integer | Can occur when there is a profile save failure |

Reviewer: ```suggestion
| Mailing Address Suggestion Failures | 23 | Hit | Integer | On profile save edit failure, this increments by 1 for mailing address specific failures |
```

> | Address Override | 24 | Hit | Integer | User does not leverage the address suggestion and instead opts to override and use their own input when submitting information on their profile |

Reviewer: Nice explanation for this one 👍

> | Screening Tool Total Results Returned | 25 | Hit | Integer | Number of results displayed on the COVID screening tool |

Reviewer: ```suggestion
| Screening Tool Total Results Returned | 25 | Hit | Integer | Increments by 1 each time a screen is completed, intended to service calculated metric "Average Screening Tool Completion Time" |
```

> | Screening Tool Total Time | 26 | Hit | Time | Time to complete the COVID screening tool |

Reviewer: ```suggestion
| Screening Tool Total Total Time | 26 | Hit | Integer | Records the total time spent completing a screen, intended to service calculated metric "Average Screening Tool Completion Time" |
```

> | BAM - Quantity Ordered | 27 | Hit | Integer | Used on a form event |

Reviewer: ```suggestion
| BAM - Quantity Ordered | 27 | Hit | Integer | Used to record the quantity of products ordered |
```

> | STEM Scholarship Criteria Met - Post 911 | 28 | Hit | Integer | Used for if the post-911 eligibility criteria on the STEM Scholarship was met. If met then convert true to `{{JS - Generic - Custom Metric Increment}}` and convert false to `{{Generic Undefined}}`.  |

Reviewer: ```suggestion
| STEM Scholarship Criteria Met - Post 911 | 28 | Hit | Integer | Increments by 1 if the post-911 eligibility criteria on the STEM Scholarship was met |
```

| STEM Scholarship Criteria Met - STEM or Teaching | 29 | Hit | Integer | Used for if the STEM or Teaching eligibility criteria on the STEM Scholarship was met. If met then convert true to `{{JS - Generic - Custom Metric Increment}}` and convert false to `{{Generic Undefined}}`.  |
| STEM Scholarship Criteria Met - Benefits Remaining | 30 | Hit | Integer | Months remaining for edu elegibility criteria use - used in the `Product Ineligibility Alerts` tag.  |

> | Disability - Claim for Spouse | 31 | Hit | Integer | Used in a disability form event to indicate if they are filing a claim for their spouse |

Reviewer: Each of these L929 - 937 are 'yes' 'no' questions in the form that increment by 1 if yes and nothing if answered no

| Disability - Step Child Left Household | 35 | Hit | Integer | Used in a disability form event for if user had a step child leave the household |
| Disability - Death of Dependent or Spouse | 36 | Hit | Integer | Used in a disability form event for if user had a death of a dependent or spouse |
| Disability - Child Married | 37 | Hit | Integer | Used in a disability form event for if user had a child get married |
| Disability - Child Stopped Attending School | 38 | Hit | Integer | Used in a disability form event for if user had a child stop attending school |
| Type Ahead Option Chosen Position | 39 | Hit | Integer | What position (integer) the type ahead option that the user selected was in relative to the list of type ahead options |
| Search Results Position | 40 | Hit | Integer | What position (integer) the search results option that the user selected was in relative to the list of search results |
| Search Total Results Returned | 41 | Hit | Integer | Number (integer) of the total results returned from a user's search |
| Search Results Total Pages Returned | 42 | Hit | Integer | Number (integer) of the total pages of search results returned from a user's search |

> | Search Results Click Page Number | 43 | Hit | Integer | If a user clicks a page number located at the bottom of the results on a search result page |

Reviewer: ```suggestion
| Search Results Click Page Number | 43 | Hit | Integer | Current pagination page number of search results click |
```

| Search Results Top Recommendations Clicks | 44 | Hit | Integer | If a user clicks a search result located in the "Top Recommendations" section of a search results page |

> | Search Performed | 45 | Hit | Integer | If a user performs a search on VA.gov |

Reviewer: ```suggestion
| Search Performed | 45 | Hit | Integer | If a user performs a search on VA.gov using [our standard search taxonomy](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/analytics/google-analytics/tracking-search.md) |
```

| Medallia Feedback Submission Value | 46 | Hit | Integer |  |
| Sitewide Search From Header Input | 47 | Hit | Integer | If a user performs a sitewide search on VA.gov via the search bar located in the header |

> | Type Ahead Suggestions Count | 48 | Hit | Integer |  |

Reviewer: ```suggestion
| Type Ahead Suggestions Count | 48 | Hit | Integer | Number of type ahead suggestions provided  |
```

> | Unused | 49 | Hit | Integer | - |

Reviewer: ```suggestion
| GIBCT Accordion Rating | 49 | Hit | Integer | Value of each GIBCT accordion ratings clicked on
```

> 

Reviewer: ```suggestion
| Component Library Event | 50 | Hit | Integer | Increments by 1 for each component library event fired 
```
